src/components/global/CardPrize.vue

```diff
@@ -0,0 +1,163 @@
+<script lang="ts">
+/**
+ * CardPrize Component
+ *
+ * @description * Use this component to show prize cards with dialog.
+ * Card is clickable and shows a dialog with detailed information.
+ *
+ * @props
+ * - `card` (CardPrize, required): The object representing a card.
+ *   It should be of type `CardPrize`.
+ *
+ * @components
+ * - `DialogDefault`: Used to display detailed information about the prize in a
+ *   modal dialog.
+ *
+ * @example
+ * <card-prize :card="card" />
+ *
+ * @see [Figma Design](https://www.figma.com/design/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?node-id=4858-104622&t=9iHoAuwC7HhXft7f-1)
+ */
+
+// libraries
+import { Screen } from 'quasar';
+import { defineComponent, ref } from 'vue';
+
+// components
+import DialogDefault from '../global/DialogDefault.vue';
+
+// types
+import type { CardPrizeType, ConfigGlobal } from '../types';
+
+// config
+const rideToWorkByBikeConfig: ConfigGlobal = JSON.parse(
```

Comment:
We have had already parsed global config inside global_vars.ts file.

Use this kind of import:

```
 import { rideToWorkByBikeConfig } from '../boot/global_vars';
```

Check all source code files, and change to this import, please (make new PR).